view/index.html

```diff
@@ -1,104 +1,174 @@
+          >
+            About Us
+          </a>
+        </li>
+      </ul>
+      <img
+        class="nav-menu-button"
+        src="../images/burger-menu.png"
+        alt="menu button"
+      />
+    </nav>
     <div class="grid-container">
-        <section class="categories-section">
-          <header id="main-title" class="homepage-header">
-            <h1>CLACKS & THOCC</h1>
-          </header>
-          <div class="categories-container">
-              <a class="category-container upper-categories feature-section" href="article-page.html">
-                  <h2 class="category-header">FEATURED</h2>
-              </a>
-              <div class="bottom-categories">
-                <a class="category-container recent-post-section" href="#articles-section">
-                  <h2 class="category-header">RECENT POSTS</h2>
-                </a>
-                <a class="category-container featured-product-section" href="https://www.gloriousgaming.com/products/glorious-gmmk-pro-75-barebone-black" target="_blank">
-                  <h2 class="category-header">SPOTLIGHT</h2>
-                </a>
-              </div>
-              
+      <section class="categories-section">
+        <header id="main-title" class="homepage-header">
+          <h1>CLACKS & THOCC</h1>
+        </header>
+        <div class="categories-container">
+          <a
+            class="category-container upper-categories feature-section"
+            href="article-page.html"
+          >
+            <h2 class="category-header">FEATURED</h2>
+          </a>
+          <div class="bottom-categories">
+            <a
+              class="category-container recent-post-section"
+              href="#articles-section"
+            >
+              <h2 class="category-header">RECENT POSTS</h2>
+            </a>
+            <a
+              class="category-container featured-product-section"
+              href="https://www.gloriousgaming.com/products/glorious-gmmk-pro-75-barebone-black"
+              target="_blank"
+            >
+              <h2 class="category-header">SPOTLIGHT</h2>
+            </a>
           </div>
-        </section>
-        <section class ="articles-container homepage-articles-section" id="articles-section">
-          <span class="articles-container-header homepage-articles-header">
-            == Check out the Newest Articles ==
-          </span>
-          <div class="article-card recent-post-card">
-            <img class="recent-post-img" src="../images/featureImage-SizeDown.jpeg" alt="two keyboard sizes">
-              <a href="article-page.html" target="_blank">
-                <h2>Down-sizing your keyboard: should you even?</h2>
+        </div>
+      </section>
+      <section
+        class="articles-container homepage-articles-section"
+        id="articles-section"
+      >
+        <span class="articles-container-header homepage-articles-header">
+          == Check out the Newest Articles ==
+        </span>
```

Comment:
I just checked the page design layout, and this element should be a header tag. Visually, it serves as the header of the Newest Articles.